spec/requests/lettings_logs_controller_spec.rb

```diff
@@ -1865,6 +1865,23 @@
         get "/lettings-logs/csv-download?codes_only=#{codes_only}", headers:, params: {}
         expect(response).to have_http_status(:unauthorized)
       end
+
+      context "when filtering by organisation and year" do
+        let(:other_organisation) { FactoryBot.create(:organisation) }
+        let(:lettings_logs) { create_list(:lettings_log, 2, :setup_completed, assigned_to: user, owning_organisation: other_organisation, managing_organisation: user.organisation) }
+
+        before do
+          create(:organisation_relationship, parent_organisation: other_organisation, child_organisation: user.organisation)
+          lettings_logs
```

Comment:
since this is the only place `lettings_logs` is called I would have a gentle preference for removing 1871 and: 
```suggestion
          create_list(:lettings_log, 2, :setup_completed, assigned_to: user, owning_organisation: other_organisation, managing_organisation: user.organisation)
```

I would even not see nay particular need to have a before block only for one test and just put it all in the it.

I feel more strongly about the first of the above and I feel very weakly about both of them

Reply:
I think it was mostly to get the year in the test`lettings_logs[0].form.start_date.year`, but now that you've mentioned it, I can probably remove it from here in general, cause it lettings_logs would get created when calling the url!